db/migrate/20181104000207_change_publication_status_to_not_null_integer.rb

```diff
@@ -0,0 +1,19 @@
+class ChangePublicationStatusToNotNullInteger < ActiveRecord::Migration[5.2]
+  def change
+    remove_column :articles, :publication_status, :string, default: 'draft'
+    remove_column :books,    :publication_status, :string, default: 'draft'
+    remove_column :journals, :publication_status, :string, default: 'draft'
+    remove_column :logos,    :publication_status, :string, default: 'draft'
+    remove_column :pages,    :publication_status, :string, default: 'draft'
+    remove_column :posters,  :publication_status, :string, default: 'draft'
+    remove_column :videos,   :publication_status, :string, default: 'draft'
+
+    add_column :articles, :publication_status, :integer, default: 'draft', null: false
```

Comment:
actually, let me think through this migration a bit...ui think it might make everything `draft` on deploy